pycm/pycm_util.py

```diff
@@ -195,30 +195,45 @@ def normalized_table_calc(classes, table):
     Return normalized confusion matrix.
 
     :param classes: classes list
-    :type classes:list
+    :type classes: list
     :param table: table
-    :type table:dict
+    :type table: dict
     :return: normalized table as dict
     """
-    normalized_table={}
+    normalized_table = {}
     for key in classes:
         normalized_table[key] = {}
         div = sum(table[key].values())
         if div == 0:
             div = 1
         for item in classes:
-            normalized_table[key][item] = numpy.around(table[key][item] / div, 5)
+            normalized_table[key][item] = custom_rounder(
+                table[key][item] / div, 5)
     return normalized_table
 
 
+def custom_rounder(input_number, digit):
+    """
+    Return round of a input number respected to the digit
+
+    :param input_number: number that should be round
+    :type input_number: float
+    :param digit: number of digits that we should round number into
+    :type digit: int
+    :return: rounded number in float
+    """
+    p = float(10**digit)
```

Comment:
This constant term will calculate in each iteration and I think it would be better if we calculate it only once and outside of this function, something like this :
```python
def custom_rounder(input_number, p):
    return int(input_number * p + 0.5) / p
```

Reply:
I'll fixed it.